group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -798,7 +961,7 @@ private void scheduleConsumerGroupSessionTimeout(
                 ConsumerGroupMember member = group.getOrMaybeCreateMember(memberId, false);
                 log.info("[GroupId {}] Member {} fenced from the group because its session expired.",
                     groupId, memberId);
-                return consumerGroupFenceMember(group, member);
+                return new CoordinatorResult<>(consumerGroupFenceMember(group, member), null);
```

Comment:
nit: There is a constructor which does not take the response. We could use it and remove `null` here. There are a few other cases. I won't mention them again.